output/influxdb/util.go

```diff
@@ -18,13 +20,21 @@ func MakeClient(conf Config) (client.Client, error) {
 	if conf.Addr.String == "" {
 		conf.Addr = null.StringFrom("http://localhost:8086")
 	}
-	return client.NewHTTPClient(client.HTTPConfig{
+	clientHTTPConfig := client.HTTPConfig{
 		Addr:               conf.Addr.String,
 		Username:           conf.Username.String,
 		Password:           conf.Password.String,
 		UserAgent:          "k6",
 		InsecureSkipVerify: conf.Insecure.Bool,
-	})
+	}
+	if conf.Proxy.Valid && conf.Proxy.String != "" {
```

Comment:
```suggestion
	if conf.Proxy.Valid {
```

`Valid` Should be enough - did you have problems with only it?

Reply:
removed